merlin/systems/dag/ops/fil.py

```diff
@@ -318,21 +279,16 @@ def export(
         version_path = node_export_path / str(version)
         version_path.mkdir(parents=True, exist_ok=True)
 
-        self.fil_model.save(version_path)
+        self.fil_model_class.save(version_path)
 
-        config = fil_config(
-            node_name,
-            self.fil_model.model_type,
-            self.fil_model.num_features,
-            self.fil_model.num_classes,
-            max_batch_size=self.max_batch_size,
-            **self.parameters,
-        )
+        return version_path
 
-        with open(node_export_path / "config.pbtxt", "w", encoding="utf-8") as o:
-            text_format.PrintMessage(config, o)
+    def load_model(self, version_path):
```

Comment:
what is this load_model required for in the context of the Triton runtimes. Since these models will be inside the FIL  backend, do we need to be able to load them for something else?

Reply:
They are not necessary for the triton side, because fil will always run in the triton custom backend. But this is not in the operator for triton, it stayed in the base operator. This will be necessary to load models when we start moving/persisting ensembles around. It is in place for the load_artifacts call that happens in non triton executors. It may be unnecessary for this PR's objective so I can remove, but I wanted to solve for this problem.